### PR TITLE
Feature/canvasses

### PR DESCRIPTION
**Pull request by dancoates**
Adds 3 basic chart canvasses:

- Line Chart
- Column Chart (allows groups of columns)
- Scatter Chart

### CODE REVIEW

File: src/component/canvas/ColumnCanvas.jsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+ *             fill: 'red'
+ *         }
+ *     ]}
+ * />;
+ *
+ *
+ */
+
+export default class ColumnCanvas extends React.PureComponent {
+    static defaultProps = {
```
Comment (dxinteractive) [inline src/component/canvas/ColumnCanvas.jsx:80]: Does this need a default prop if it's a required prop?

File: src/component/canvas/ScatterCanvas.jsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+ * @prop {Scale} scaleY
+ * Any d3-scale for the y axis
+ *
+ * @prop {string} columnX
+ * The column key from `ChartData` to use for the x axis
+ *
+ * @prop {string} columnY
+ * The column key from `ChartData` to use for the y axis
+ *
+ * @prop {Dot} [dot]
```
Comment (dxinteractive) [inline src/component/canvas/ScatterCanvas.jsx:65]: Can do `[dot] = <Circle />` here to make the default value show up in the prop summary.

**Review by dxinteractive — CHANGES_REQUESTED** [review]

File: src/component/canvas/ColumnCanvas.jsx (hunk 1 above)
Comment (dancoates) [reply src/component/canvas/ColumnCanvas.jsx:80]: Ah yeah, it shouldn't be required.

File: src/component/canvas/ScatterCanvas.jsx (hunk 2 above)
Comment (dancoates) [reply src/component/canvas/ScatterCanvas.jsx:65]: The description was a bit unclear but it doesn't actually default to a `<circle/>` element, it defaults to a wrapper component that then renders a circle. I've updated the wording to hopefully be a bit more clear.

**Review by allanhortle — APPROVED** [review]

**Review by dxinteractive — APPROVED** [review]

**Pull request merged**